lib/Dialect/Moore/MooreTypes.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "circt/Dialect/Moore/MooreTypes.h"
 #include "circt/Dialect/Moore/MooreDialect.h"
+#include "circt/Support/LLVM.h"
```

Comment:
nit: This may be an unnecessary include.

Reply:
Sorry I will be careful next time